src/features/feedback/components/TeamFeedbackList.tsx

```diff
@@ -1,19 +1,14 @@
 import { Box, Stack, StackDivider } from '@chakra-ui/react'
 import { useFilterFeedback } from '../api/useFilterFeedback'
-import { TeamFeedbackListSkeleton } from './TeamFeedbackListSkeleton'
 import { TeamFeedbackRow } from './TeamFeedbackRow'
 
 export const TeamFeedbackList = (): JSX.Element => {
-  const { filteredFeedback, isFetching } = useFilterFeedback()
-
-  if (!filteredFeedback) {
-    return <TeamFeedbackListSkeleton />
```

Comment:
should we use this skeleton as a suspense fallback instead of removing it completely?

Reply:
Skeleton is still used, please refer to `DashboardContainer` wrapper component within `Dashboard.tsx`